runtime/moonriver/tests/integration_test.rs

```diff
@@ -1042,34 +1003,9 @@ fn revoke_nomination_via_precompile() {
 				None, // Use the next nonce
 			))
 			.dispatch(<Runtime as frame_system::Config>::Origin::root()));
-
+			run_to_block(600);
```

Comment:
The only change for most of these tests is waiting 2 rounds to check if exit executed?

Reply:
Correct